src/components/dialogs/parameters/single-line-diagram-parameters.js

```diff
@@ -57,89 +55,43 @@ export const SingleLineDiagramParameters = ({
-                            value={SubstationLayout.SMARTHORIZONTALCOMPACTION}
-                        >
-                            <FormattedMessage id="SmartWithHorizontalCompactionSubstationLayout" />
-                        </MenuItem>
-                        <MenuItem
-                            value={SubstationLayout.SMARTVERTICALCOMPACTION}
-                        >
-                            <FormattedMessage id="SmartWithVerticalCompactionSubstationLayout" />
-                        </MenuItem>
-                    </Select>
-                </Grid>
+                <ParamLine
+                    type={ParameterType.DropDown}
+                    param_name_id={PARAM_SUBSTATION_LAYOUT}
+                    labelTitle="SubstationLayout"
+                    labelValue="substation-layout-select-label"
+                    defaultValueIfNull={true}
+                    values={{
+                        [SubstationLayout.HORIZONTAL]:
+                            'HorizontalSubstationLayout',
+                        [SubstationLayout.VERTICAL]: 'VerticalSubstationLayout',
+                        [SubstationLayout.SMART]: 'SmartSubstationLayout',
+                        [SubstationLayout.SMARTHORIZONTALCOMPACTION]:
+                            'SmartWithHorizontalCompactionSubstationLayout',
+                        [SubstationLayout.SMARTVERTICALCOMPACTION]:
+                            'SmartWithVerticalCompactionSubstationLayout',
+                    }}
+                />
                 <LineSeparator />
-                <Grid item xs={8}>
-                    <Typography component="span" variant="body1">
-                        <Box fontWeight="fontWeightBold" m={1}>
-                            <FormattedMessage id="ComponentLibrary" />
-                        </Box>
-                    </Typography>
-                </Grid>
-                <Grid item container xs={4} className={classes.controlItem}>
-                    <Select
-                        size="small"
-                        labelId="component-library-select-label"
-                        value={
-                            componentLibraryLocal !== null
-                                ? componentLibraryLocal
-                                : componentLibraries[0]
-                        }
-                        onChange={(event) => {
-                            handleChangeComponentLibrary(event.target.value);
-                        }}
-                    >
-                        {componentLibraries.map((library) => {
-                            return (
-                                <MenuItem key={library} value={library}>
-                                    {library}
-                                </MenuItem>
-                            );
-                        })}
-                    </Select>
-                </Grid>
+                <ParamLine
+                    type={ParameterType.DropDown}
```

Comment:
missing defaultValueIfNull ? Or did we change the behavior on purpose ? Haven't checked the other but for me componentLibrary was not set and before the app showed a value, but now it's empty